### PR TITLE
add weui v0.3.0

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/weui/weui
**Watch 624
  Star 6,106
 Fork 1,500**
@Amomo could u help me check this PR for #6777 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [ ] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/weui/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/weui/weui.git"
+  },
+  "homepage": "https://github.com/weui/weui",
+  "license": "MIT",
+  "npmName": "weui",
+  "npmFileMap": [
+    {
+      "basePath": "dist/style",
```
Comment (PeterDaveHello) [inline ajax/libs/weui/package.json]: Should use `dist` as basePath.

File: ajax/libs/weui/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "weui",
+  "version": "0.3.0",
+  "description": "为微信 Web 服务量身设计",
```
Comment (PeterDaveHello) [inline ajax/libs/weui/package.json]: The desc of the repo is better:
`A UI library by WeChat official design team, includes the most useful widgets/modules in mobile web applications.`

**Comment by progrape** [comment]
@maruilian11 @PeterDaveHello thanks for provide cdn. We have updated the description of WeUI.

https://github.com/weui/weui/blob/master/package.json#L4

**Comment by PeterDaveHello** [comment]
@progrape nice

**Comment by maruilian11** [comment]
@Amomo i have fixed `description` and `basepath`, could help me you check this PR? thanks!

**Comment by PeterDaveHello** [comment]
@maruilian11 you forgot you rebase your branch with our latest master branch.

**Pull request merged**